pom.xml

```diff
@@ -13,7 +13,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <surefire.version>2.20</surefire.version>
+        <surefire.version>3.0.0-M5</surefire.version>
```

Comment:
I don't think we should be using the milestone versions of anything.

Is 3.0.0 available?

Reply:
The surefire plugin v3.x has only been on milestone releases since 2018:
http://archive.apache.org/dist/maven/surefire/

But it builds fine with 2.22-2, and Java 11 support was added in 2.21.

Reply:
that's odd, maybe it's not supported anymore